pkg/analyzer/analyzer.go

```diff
@@ -44,10 +42,9 @@ func init() {
 }
 
 var Analyzer = &analysis.Analyzer{
-	Name:     "gci",
-	Doc:      "A tool that control golang package import order and make it always deterministic.",
-	Requires: []*analysis.Analyzer{inspect.Analyzer},
```

Comment:
Why remove this?

Reply:
It is not used by the analyzer at all, but does some work

I tested and did not find any problems